src/test/container-features/configs/invalid-configs/dockerfile-with-v2-local-features-no-dev-container-folder/local-features/localFeatureA/install.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+echo "Activating feature 'localFeatureA'"
+
+GREETING=${GREETING:-undefined}
+echo "The provided greeting is: $GREETING"
+
+tee /usr/hello.sh > /dev/null \
```

Comment:
See feature-template for simplified version. I suggest to copy that around for clarity.